tests/languages/docker_test.py

```diff
@@ -12,3 +16,75 @@ def invalid_attribute():
         getgid=invalid_attribute,
     ):
         assert docker.get_docker_user() == ()
+
+
+class TestInDocker:
+    @mock.patch('builtins.open', new_callable=mock_open)
+    def test_in_docker_no_file(self, mock_file):
+        mock_file.side_effect = FileNotFoundError
+
+        assert docker.is_in_docker() is False
+        mock_file.assert_called()
+
+    @mock.patch('builtins.open', new_callable=mock_open, read_data='tdockert')
+    def test_in_docker_docker_in_file(self, _):
+        assert docker.is_in_docker() is True
+
+    @mock.patch('builtins.open', new_callable=mock_open, read_data='testtest')
+    def test_in_docker_docker_not_in_file(self, _):
+        assert docker.is_in_docker() is False
+
+
+class TestDockerPath:
+    @mock.patch.object(docker, 'is_in_docker', return_value=False)
+    def test_not_in_docker_returns_same(self, _):
+        assert docker.get_docker_path('abc') == 'abc'
+
+    @mock.patch.object(docker, 'is_in_docker', return_value=True)
+    def test_in_docker_no_binds_same_path(self, _):
+        binds_list: List[str] = []
```

Comment:
if you inline this you shouldn't need the type signature (?)

Reply:
I got warning from CI about it

Reply:
I mean if you don't have this as a variable you shouldn't need the annotation

Reply:
I want to have consistent tests so they look alike